### PR TITLE
chore: Add Kubeflow Steering Committee members to the org admin role.

### PR DESCRIPTION
**Pull request by zijianjoy**
As title described.

### CODE REVIEW

File: github-orgs/kubeflow/org.yaml [hunk 1]
```diff
@@ -5,12 +5,16 @@
...
         - caniszczyk
         - chensun
         - google-oss-robot
         - googlebot
         - james-jwu
+        - jbottum
+        - johnugeorge
         - krook
+        - terrytangyuan
         - theadactyl
```
Comment (terrytangyuan) [inline github-orgs/kubeflow/org.yaml:18]: We can remove Thea since she’s no longer involved

Comment (zijianjoy) [reply github-orgs/kubeflow/org.yaml]: Done.

File: github-orgs/kubeflow/org.yaml [hunk 2]
```diff
@@ -5,13 +5,16 @@
 orgs:
     kubeflow:
         admins:
+        - andreyvelich
         - caniszczyk
```
Comment (james-jwu) [inline github-orgs/kubeflow/org.yaml:9]: Should we remoe caniszczyk and krok?

Comment (zijianjoy) [reply github-orgs/kubeflow/org.yaml:9]: Referring to https://github.com/kubeflow/internal-acls/pull/646, they are required to be added by the CNCF.

**Pull request merged**